js/data/line_bucket.js

```diff
@@ -105,6 +136,10 @@ LineBucket.prototype.addLine = function(vertices, join, cap, miterLimit, roundLi
         // of the segments between the previous line and the next line.
         var joinNormal = prevNormal.add(nextNormal)._unit();
 
+        if (isNaN(joinNormal.x) || isNaN(joinNormal.y)) {
+            return;
+        }
+
```

Comment:
Oops! Just some innocuous-looking leftover debugging code. Will remove in the next commit.